core-web/libs/edit-content/src/lib/fields/dot-edit-content-json-field/dot-edit-content-json-field.component.scss

```
@@ -0,0 +1,17 @@
+@use "variables" as *;
+
+.json-field__container {
+    min-height: 9.375rem; // 150px
+    height: 100%;
+    width: 100%;
+    overflow: auto;
+}
+
+.json-field__editor {
```

```suggestion
ngx-monaco-editor {
```

Is only one... just saying :)

---

Done!